src/main/shell-session/shell-session.ts

```diff
@@ -50,23 +51,19 @@ export abstract class ShellSession {
   protected kubectlBinDirP: Promise<string>;
   protected kubeconfigPathP: Promise<string>;
 
-  protected get cwd(): string | undefined {
```

Comment:
Probably making this an `abstract get cwd()` would be a good, then we don't need to change the arguments of `open()`

Reply:
Makes much more sense, will update.